src/store/metadata/oxigraph/store.py

```diff
@@ -306,6 +339,11 @@ def get_version(
             del column["@id"]
         version_graph["table_schema"]["columns"] = columns_graph
         del version_graph["table_schema"]["@id"]
+
+        if type(version_graph["issued"]) != str:
```

Comment:
you can remove this check and logging, made sense at the time but we've had some issues with data so we're going to push the data validation upstream so its always right when it gets to us, since its right it'll have exactly one issued date.